docs/cn/api/POSIX-API.md

```diff
@@ -127,6 +129,26 @@ Alluxio-FUSE是基于标准的alluxio-core-client-fs进行操作的。你也许
 
 一种方法是编辑`$ALLUXIO_HOME/conf/alluxio-site.properties`配置文件来更改客户端选项。注意所有的更改应该在Alluxio-FUSE启动之前完成。
 
+### 选择libfuse的版本
+
+Alluxio现在支持libfuse2和libfuse3。
```

Comment:
talk a bit about why people may choose between libfuse 2 & 3?
e.g., my guess is libfuse 2 is more stable and more tested in production; while libfuse3 with Alluxio is experimental, though it may provide higher performance or more functions?
otherwise, users have no clue to make a decision here according to the manual

Reply:
updated